### ISSUE TITLE
Invalid token when run bean-check

### ISSUE THREAD

**Issue by ziyunch** [Akuukis/beancount_share#2]
Hi,

Thank you for developing this tool. I just installed the plugin by `pip3 install beancount_share --user`. However, when I put the following into my main.bean file and run `bean-check main.bean` I got the following errors:

```beancount
plugin "beancount_share" "{
    mark_name: "share",
    meta_name: "None",
    account_debtors: "Liabilities:Receivables",
    account_creditors: "Liabilities:Receivables",
    open_date: "None",
    quantize: "0.01"
}"
```

> main.bean:73:      Invalid token: 'share",'
> main.bean:79:      Invalid token: '"'

Thanks!

**Comment by ziyunch** [comment]
In addition, even though I use the default setting `plugin "beancount_share" "{}"`, it won't transform my tagged transaction.

**Comment by Akuukis** [comment]
Oops, there's a mistake in readme.

Please use single-quotes instead of double-quotes, otherwise beancount thinks the config ends and the first double-quote instead. Also, use `None` without quotes like Python does. So it would look like this:
```
plugin "beancount_share" "{
    mark_name: 'share',
    meta_name: None,
    account_debtors: 'Liabilities:Receivables',
    account_creditors: 'Liabilities:Receivables',
    open_date: None,
    quantize: '0.01'
}"
```

**Comment by Akuukis** [comment]
Fixed at 241e2f36faf5225ee8bdd38fa9bbd57384cfcb3c & published

**Comment by ziyunch** [comment]
Thanks! This works for me.

Sure, created a separate issue https://github.com/Akuukis/beancount_share/issues/3
